dhis-2/dhis-services/dhis-service-core/src/test/java/org/hisp/dhis/dataapproval/DataApprovalServiceTest.java

```diff
@@ -1000,8 +994,8 @@ public void testGetDataApprovalStateOtherPeriodTypes()
         assertEquals( DataApprovalState.UNAPPROVED_READY, dataApprovalService.getDataApprovalStatus( workflow1234, periodW, organisationUnitD, defaultOptionCombo ).getState() );
     }
 
+    @Ignore
```

Comment:
Do you remember why this is ignored? Is there a Jira issue for fixing it?

Reply:
Unignored.